app/assets/stylesheets/application.scss

```diff
@@ -30,7 +30,18 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+ .post-it {
+ background:#fefabc;
```

Comment:
Color literals like `#fefabc` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space